core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -233,11 +259,16 @@ export class OptimizedSsrEngine {
     options: any,
     callback: SsrCallbackFn
   ): void {
+    const render = { uuid: randomUUID(), timeStart: new Date().toISOString() };
+    options.req.res.locals = { cx: { render } };
```

Comment:
[MEDIUM]

As we discussed offline, those values (uuid and timeStart) are bound to a request, but not a render.
What about renaming `render` to `request`? and `timeStart` to `timeReceived`?